docs/simulations.rst

```diff
@@ -2,27 +2,41 @@
 ###########
 
-Simulations of observations are useful for testing the sensitivity
-of a specific set of observations to BEAST model parameters.  This is
-useful for quantifying the BEAST performance for proposed and actual
-observations.
-
-This is done using the
-`beast.observationmodel.observations.gen_SimObs_from_sedgrid` function.
-The script
-`tools/simulate_obs.py` can be run directly or using the `beast simulate_obs`
-command once the beast has been installed.
+Simulations of observations can be made based on already computed  physics and
+observation model grids.  Uses for such simulations include testing the
+sensitivity of a specific set of observations to BEAST model parameters and
+testing ensemble fitters like the MegaBEAST.
+
+Simulations are done using the
+`beast.observationmodel.observations.gen_SimObs_from_sedgrid` function. The
+script `tools/simulate_obs.py` provides a command line interface and can be run
+using the `beast simulate_obs` command once the beast has been installed.
 Simulations require already created BEAST physics and observation model grids.
-The physics model grid includes the ensemble parameters as these are the same
-as the BEAST :ref:`beast_priors`.
-If a different ensemble model is needed (e.g., with a different SFH), then a
-new physics model (and possible observations model) will be needed.
-The module
-uses already created BEAST physics and observation model grids
-by sampling the full nD prior function that is part of the physics
-model grid.  The observation model grid provides the information on
+The physics model grid includes the ensemble parameters as these are the same as
+the BEAST :ref:`beast_priors`. If a different ensemble model is needed (e.g.,
+with a different SFH), then a new physics model (and possible observations
+model) will be needed. The module uses already created BEAST physics and
+observation model grids by sampling the full nD prior function that is part of
+the physics model grid.  The observation model grid provides the information on
 the flux uncertainties and biases as well as the completeness.
 
+The number of observations to simulate is either calculated from the age and
+mass prior/ensemble models or can be explicitly specified by using the `nsim`
+parameter.
+
+If the `beastinfo` ASDF file is passed to the `simulate_obs` script,
+then the number of stars is computed based on the age/mass models.  This is
+done by computing the total mass formed by integrating the star formation
+rate versus time (age prior) multiplied by the total mass formed per unit time
+(mass prior).  This number of total stars formed is then the total mass divided
+by the average mass computed from the mass prior model.  This total number of
+stars is simulated and then the completeness function is applied to remove
+all the simulated stars that would not be observed.  This provides
+
+the star formation rate verus time in the age prior combined with the
+
+is given by the `--nsim` parameter.
```

Comment:
Woops, forgot to add that this has a few extra line breaks!

Reply:
(which I also just realized was in @kmcquinn's review, too)

Reply:
Fixed.